.github/workflows/ci.yaml

```diff
@@ -7,20 +7,6 @@ env:
   RUSTDOCFLAGS: -D warnings
 
 jobs:
-  Features:
```

Comment:
Removing features as it was causing issues since live.rs uses the same options as prerecorded_from_url.rs for non streaming features